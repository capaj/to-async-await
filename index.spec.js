import toAsyncAwait from './index'
import test from 'ava'
import fs from 'fs'

test('simplest', t => {
  const simplest = fs.readFileSync('./fixtures/simplest.js', 'utf8')
  t.snapshot(toAsyncAwait(simplest))
})

test.only('basic', t => {
  const basic = fs.readFileSync('./fixtures/basic.js', 'utf8')
  t.snapshot(toAsyncAwait(basic))
})
