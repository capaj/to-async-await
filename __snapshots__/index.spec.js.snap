// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic 1`] = `
"let a;

module.exports = () => {
  var _temp;

  const c = await a.then((b) => {
    return b + 1
  });

  return c + 1;
};"
`;
